webapp/cypress/component/SampleTableTest.cy.jsx

```diff
@@ -28,19 +28,79 @@ describe("SampleTable Component Tests", () => {
               name: "Sample 1",
               chemform: "H2O",
               date: "2023-09-01T12:34:56Z",
-              collections: ["collection1"],
+              collections: [{ collection_id: "collection1" }],
               creators: [{ display_name: "Creator 1" }],
               nblocks: 1,
+              nfiles: 1,
+              blocks: [{ title: "NMR" }],
+            },
+            {
+              item_id: "sample2",
+              type: "samples",
+              name: "Sample 2",
+              chemform: "H2O",
+              date: "2023-09-02T12:34:56Z",
+              collections: [{ collection_id: "collection2" }],
+              creators: [{ display_name: "Creator 2" }],
+              nblocks: 2,
+              nfiles: 2,
+              blocks: [{ title: "NMR" }, { title: "insitu" }],
```

Comment:
I think these should perhaps be looking for `blocktype: 'nmr'` rather than the title since the titles are editable by users, but maybe that's for another PR...